### ISSUE TITLE
[Novo spider base]: imap

### ISSUE THREAD

**Issue by trevineju** [okfn-brasil/querido-diario#1109]
### URL

https://imap.org.br/

### Municípios replicados

```bash
1. https://dom.imap.org.br/sitesMunicipios/imprensaOficial.cfm?varCodigo=133
2. https://dom.imap.org.br/sitesMunicipios/imprensaOficial.cfm?varCodigo=497
3. https://dom.imap.org.br/sitesMunicipios/imprensaOficial.cfm?varCodigo=571
```

### Formato da publicação

Os diários oficiais são publicados em PDF (texto)

### Detalhes do site de publicação

Possui filtro por data

### Observações

Para mais informações, consulte a documentação sobre [sistemas replicáveis](https://docs.queridodiario.ok.org.br/pt-br/latest/contribuindo/raspadores.html#as-classes-sistemagazettespider).

### Não existe uma issue aberta sobre isso

- [X] Eu verifiquei e não existe uma issue aberta ou marcada com <a href='https://github.com/okfn-brasil/querido-diario/issues?q=label%3Aincompatible+is%3Aclosed'>*incompatible*</a> para o mesmo sistema.

**Comment by victorfernandesraton** [comment]
> Esta issue endereça também um problema de manutenção para o município de Maragogipe-BA. Este é o município cuja URL tem o código 497.

Este município está sendo atendido pelo SAI não?

**Comment by trevineju** [comment]
> Este município está sendo atendido pelo SAI não? 

sim! Este município foi resolvido realmente! Obrigada pelo comentário

> Pelo que vi há mais diários oficiais neste imap do que no próprio SAI, coisas como diários oficiais do ano de 2011, coisa que não é suportada pelo SAI

SAI começa em 02/02/2011, assim como a versão do site no IMAP. 
- SAI: https://sai.io.org.br/ba/maragojipe/site/diariooficial
- IMAP: https://dom.imap.org.br/sitesMunicipios/imprensaOficial.cfm?varCodigo=497

Então, por esse lado, sussa.
